src/app/item-popup/ItemSocketsGeneral.tsx

```diff
@@ -72,9 +72,16 @@ function ItemSocketsGeneral({
     return null;
   }
 
-  const exoticArmorPerk = item.sockets.categories.find(
-    (c) => c.category.hash === SocketCategoryHashes.ArmorPerks_LargePerk
-  )?.sockets[0];
+  const exoticPluggable =
+    item.bucket.inArmor &&
+    item.sockets?.allSockets.find((socket) =>
+      socket.plugged?.plugDef.plug.plugCategoryIdentifier.match(/enhancements\.exotic/i)
```

Comment:
not sure if is should do `.test(/enhancements\.exotic/i)` or `.includes("enhancements.exotic")` instead.